include/filewriter/util.hpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file util.hpp
```

Comment:
In https://github.com/mlpack/mlpack/pull/2400 @birm modified each file to use the full path, for this file this would mean `@file filewriter/util.hpp` might be a good idea to do the same here as well, what do you think?